controllers/psychologists.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs')
+const Psychologists = require("../data/psychologists.json")
+
+const getPsychologists = (req, res) => {
+  res.status(200).json(Psychologists)
+}
+
+const createPsychologist = (req, res) => {
+  const params = req.query
+
+  const newPsychologist = {
+      id: new Date().getTime().toString(),
+      first_name: params.first_name,
+      last_name: params.last_name,
+      phone: params.phone,
+      email: params.email,
+      availability: JSON.parse(req.query.availability),
+      address: params.address,
+  }
+
+  Psychologists.push(newPsychologist)
+
+  fs.writeFile('./data/psychologists.json', JSON.stringify(Psychologists), {}, (error) => {
+    if(error) {
+        res.status(400).send(error)
+    } else {
+        res.status(201).json(newPsychologist)
+    }
```

Comment:
just return each `res.status(...` to avoid using an else condition.

  ```fs.writeFile('./data/psychologists.json', JSON.stringify(Psychologists), {}, (error) => {
    if(error) {
        return res.status(400).send(error)
    }
    return res.status(201).json(newPsychologist)
 
```